concent-vm/helper-scripts/concent-reset.sh

```diff
@@ -0,0 +1,43 @@
+#! /bin/bash -e
+
+# Confirm executing script
+read -p "Are you sure about execute this script? It's destroy databases, queues and other components like: Geth etc. [y/n]" -n 1 -r
```

Comment:
Please move `-n 1 -r` to the beginning of the line so that they're more visible.

Reply:
Done.